app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -41,8 +41,8 @@ def body
         def handle_http_response(http_response)
           LexisNexis::Responses::TrueIdResponse.new(
             http_response,
-            liveness_checking_required,
             config,
+            liveness_checking_required,
```

Comment:
I see - this change is needed [because of the argument order here](https://github.com/18F/identity-idp/blob/abe69830bdb4cfb08ad9a90d101566084a26a40e/app/services/doc_auth/lexis_nexis/responses/true_id_response.rb#L47).

Thanks for fixing this!